src/docs/releases/1.7.0.md

```diff
@@ -87,3 +87,6 @@ Due to Facebook's recent rebranding, all of the Facebook features have been rena
 
 Due to Microsoft's recent rebranding, Azure AD has been renamed to Microsoft Entra ID. This only concerns UI labels and documentation, but technical identifiers remain, so you shouldn't expect anything breaking.
 
+### `OrchardCore.OpenId` Module
+
+The `OpenId` module is enabling `plain` code challenge method by default, instead of `S256` that is previously used.
```

Comment:
The wording is not right at all as it implies `S256` is no longer used, which is absolutely not the case. See https://github.com/OrchardCMS/OrchardCore/pull/14013#issuecomment-1638328259.

Reply:
What do I mean that `S256` is previously used by default? If you need to rephrase please write a change suggestion

Reply:
Enabled by default before the PR: `S256`.
Enabled by default after the PR: `plain`, `S256` (`S256` remains the recommended choice).

> If you need to rephrase please write a change suggestion

Honestly, there's no point adding that to the changelog, no one will notice this change 😄

Reply:
We prefer to do this, even the naming, for instance, Azure AD -> Microsoft Entra AD, this will help the users to track what changed from one release to another